web_src/css/markup/codepreview.css

```diff
@@ -0,0 +1,26 @@
+.markup .code-preview-container {
+  border: 1px solid var(--color-secondary);
+}
+
+.markup .code-preview-container .code-preview-header {
+  border-bottom: 1px solid var(--color-secondary);
+  padding: 0.5em;
+  font-size: 11px;
```

Comment:
Normally we render code at 12px. Maybe create a root variable `--font-size-code: 12px`?

Reply:
I do not think it is necessary because it is only for the "header". There are already a lot of 12px:

![image](https://github.com/go-gitea/gitea/assets/2114189/4dea4690-0732-436d-ab48-c8b7d345720d)

Reply:
```suggestion
  font-size: 12px;
```